File: cafeapi/Models/cafeDto.cs
namespace cafeapi.Models;

public record CafeDto(
    string Id,
    string Name,
    string Address,
    string Description,
    double StudyScore, // Will be calc'd with average
    // Features from Googles API
    bool HasWifi, 
    bool IsQuiet,
    bool HasOutlets,
    Coordinates Coordinates,
    string ImageUrl
);

public record Coordinates(double Lat, double Lng);

File: client/src/types/cafedto.ts
export interface cafeDTO {
    id: string;
    name: string;
    address: string;
    description: string;
    studyScore: number;
    hasWifi: boolean;
    isQuiet: boolean;
    hasOutlets: boolean;
    coordinates: coordinates;
    imageUrl: string;
}

export interface coordinates {
    lat: number;
    lng: number;
}